index.html

```diff
@@ -165,10 +165,19 @@ <h2 class="heading-36"> About us </h2>
             <!-- Add Column 1 here / TODO -->
             Column 1
           </div>
-          <div id="about-part-2">
-            <!-- Add column 2 here / TODO -->
-            Column 2
+          
+          <div class="Column-2">
```

Comment:
Keep it's id as id="about-part-2", don't remove it and instead don't use Column-2.